src/components/Emoji.vue

```diff
@@ -1,7 +1,9 @@
 <template>
-  <span
+  <component
+    :is="tag"
```

Comment:
TODO: update docs, now we can specify `tag` for `Emoji` component. The `span` is useful to just show (non-clickable) emoji and `button` is useful for clickable emojis like those we have in the picker.

Note: this does not affect the picker as I do not use the `Emoji` component there (a performance fix, the regular tag is used there instead).